uwblueprint/community-fridge-kw
# refactor: Reuse scheduling components

## Notion ticket link
Closes 
[Make Scheduling Flow Resuable](https://www.notion.so/uwblueprintexecs/Update-Scheduling-Flow-to-be-reusable-783e7486db734c898b257dea9345aa08)
[Confirm Details Screen](https://www.notion.so/uwblueprintexecs/Scheduling-Flow-Confirm-Details-641d2b0038d14f58ac32c132c9e11f9f)



## Implementation description
* Made scheduling components to be reusable so that editing donation would use the same components
Styling still needs to be worked on



## Steps to test
1. Check normal creating a donation flow. You can use the preview url and use the seeded user to login: email: test@test.org and pw: password123
2. Edit a scheduled donation which should take to the confirm donation details screen in the scheduling flow
3. Edit buttons on donation details screen should take the user to the respective screen that they are trying to edit


https://user-images.githubusercontent.com/37384882/142585100-ea436908-98d3-4bbc-979b-0827b2c0fa4c.mov




## What should reviewers focus on?
* 


## Checklist
